testsuite/MDAnalysisTests/formats/test_libdcd.py

```diff
@@ -0,0 +1,362 @@
+from __future__ import print_function
+
+from nose.tools import raises
+from numpy.testing import assert_equal, assert_array_equal
+from numpy.testing import assert_array_almost_equal
+from numpy.testing import assert_almost_equal
+
+from MDAnalysis.lib.formats.libdcd import DCDFile
+from MDAnalysisTests.datafiles import (PSF, DCD, DCD_NAMD_TRICLINIC,
+                                       PSF_NAMD_TRICLINIC)
+
+from unittest import TestCase
+import MDAnalysis
+from MDAnalysisTests.tempdir import run_in_tempdir
+from MDAnalysisTests import tempdir
+import numpy as np
+import os
+
+class DCDReadFrameTest(TestCase):
+
+    def setUp(self):
+        self.dcdfile = DCDFile(DCD)
+        self.natoms = 3341
+        self.traj_length = 98
+        self.new_frame = 91
+        self.context_frame = 22
+        self.num_iters = 3
+        self.expected_remarks = '''* DIMS ADK SEQUENCE FOR PORE PROGRAM                                            * WRITTEN BY LIZ DENNING (6.2008)                                               *  DATE:     6/ 6/ 8     17:23:56      CREATED BY USER: denniej0                '''
+
+    def tearDown(self):
+        del self.dcdfile
+
+    def test_header_remarks(self):
+        # confirm correct header remarks section reading
+        with self.dcdfile as f:
+            list_chars = []
+            for element in f.remarks:
+                list_chars.append(element)
```

Comment:
`list_chars` is never used for a check. Why not use a direct string equality check?